test/scripts/withdraw.js

```diff
@@ -433,6 +436,25 @@ contract("Withdraw script", function (accounts) {
       assert.equal(usdcMasterBalance, toErc20Units("10000", 6).toString(), "Master safe did not receive USDC")
       assert.equal(wethMasterBalance, toErc20Units("50", 18).toString(), "Master safe did not receive WETH")
     })
+    it("combines withdraw and transfer ignoring bracket token balance", async () => {
+      const amounts = [{ tokenData: { decimals: 18, symbol: "WETH" }, amount: "0" }]
+      const [masterSafe, bracketAddresses, tokenInfo] = await setup(2, amounts)
+      const wethToken = await MintableToken.at(tokenInfo[0].address)
+      await wethToken.mint(bracketAddresses[0], toErc20Units("10", 18))
+
+      // no withdraw requested
+      const argv = {
+        masterSafe: masterSafe.address,
+        brackets: bracketAddresses,
+        tokens: [tokenInfo[0].address],
+        withdraw: true,
+        transferFundsToMaster: true,
+      }
+      const transaction = await prepareWithdraw(argv)
+      // if the built transaction used the token balance of the bracket instead of zero, then
+      // the following transaction would fail.
+      await await execTransaction(masterSafe, safeOwner.privateKey, transaction)
```

Comment:
one await too much

Reply:
```suggestion
      await execTransaction(masterSafe, safeOwner.privateKey, transaction)
```

Reply:
Code suggestions! How do you do that?

Reply:
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

Point 6